src/test/java/com/amazon/opendistroforelasticsearch/security/compliance/ComplianceConfigTest.java

```diff
@@ -0,0 +1,175 @@
+package com.amazon.opendistroforelasticsearch.security.compliance;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import com.google.common.cache.LoadingCache;
+import org.joda.time.ReadableInstant;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.ArrayList;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
+public class ComplianceConfigTest {
+
+    @Test
+    public void test() throws InvocationTargetException {
+        ComplianceConfig objectUnderTest = (ComplianceConfig) Reflector.getInstance(
+                "com.amazon.opendistroforelasticsearch.security.compliance.ComplianceConfig");
+        objectUnderTest.setExternalConfigLogged(false);
+        Reflector.setField(objectUnderTest, "logExternalConfig", false);
+
+        //for extra coverage calling getters
+        Assert.assertNull(objectUnderTest.getSettings());
+        Assert.assertNull(objectUnderTest.getEnvironment());
+
+        Assert.assertFalse(objectUnderTest.isLogExternalConfig());
+        Assert.assertFalse(objectUnderTest.isExternalConfigLogged());
+        Assert.assertFalse(objectUnderTest.isEnabled());
```

Comment:
What does this test?

Reply:
//for extra coverage calling getters
I have removed them